yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -559,6 +561,10 @@ func createMigrationAssessmentCompletedEvent() *cp.MigrationAssessmentCompletedE
 func convertAssessmentIssueToYugabyteDAssessmentIssue(ar AssessmentReport) []AssessmentIssueYugabyteD {
 	var result []AssessmentIssueYugabyteD
 	for _, issue := range ar.Issues {
+		extraDetails, err := json.Marshal(issue.Details)
+		if err != nil {
+			utils.ErrExit("error marshalling details: %v", err)
+		}
```

Comment:
couple of points:
1. No ErrExit in middle of code.
2. Why do we have to marshal it? see if there can be a better way?

Reply:
No, the field Details in the `AssessmentYugabyteD` is the `json.RawMessage` we can't directly put map in that so need marshal for it. I am not sure if we have intentionally added that type on this field, so didn't changed it. cc: @makalaaneesh @sanyamsinghal

Reply:
Made this change, let me know if there are any concerns